### PR TITLE
Integrate Shell runtime

### PR DESCRIPTION
**Pull request by jasl**
Beware:

- I don't know we should handle <https://github.com/paritytech/cumulus/blob/master/polkadot-parachains/src/service.rs#L936-L954> 
- There's no `sudo` nor `balance` in Shell runtime, though we can integrate them

UPDATE：

- Introduced `sudo`

### CODE REVIEW

**Comment by h4x3rotab** [comment]
We need sudo in the runtime. Balance is not necessary though

**Comment by h4x3rotab** [comment]
Also we need to test if we can send sudo transaction on the parachain

File: runtime/shell/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,405 @@
...
+	type ReservedDmpWeight = ReservedDmpWeight;
+	type XcmpMessageHandler = ();
+	type ReservedXcmpWeight = ();
+}
+
+impl parachain_info::Config for Runtime {}
+
+parameter_types! {
+	pub const RococoLocation: MultiLocation = MultiLocation::parent();
+	pub const RococoNetwork: NetworkId = NetworkId::Polkadot;
```
[the remark this answers is not in the document]
Comment (jasl) [reply runtime/shell/src/lib.rs:178]: this is from original Shell runtime <https://github.com/paritytech/cumulus/blob/polkadot-v0.9.16/polkadot-parachains/shell/src/lib.rs#L176-L180>
do you think we should remove this?

[the remark this answers is not in the document]
Comment (h4x3rotab) [reply runtime/shell/src/lib.rs:178]: Not a big deal. Can leave there

File: runtime/shell/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,405 @@
...
+use xcm_builder::{
+	AllowUnpaidExecutionFrom, FixedWeightBounds, LocationInverter, ParentAsSuperuser,
+	ParentIsDefault, SovereignSignedViaLocation,
+};
+use xcm_executor::{Config, XcmExecutor};
+
+/// This runtime version.
+#[sp_version::runtime_version]
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+	spec_name: create_runtime_str!("shell"),
```
[the remark this answers is not in the document]
Comment (jasl) [reply runtime/shell/src/lib.rs]: changed to `khala`

**Pull request merged**